### PR TITLE
add katello_content_view_filter module

### PR DESCRIPTION
**Pull request by sean797**
Requires:
~~https://github.com/SatelliteQE/nailgun/pull/504~~
~~https://github.com/SatelliteQE/nailgun/pull/505~~
~~https://github.com/SatelliteQE/nailgun/pull/506~~
~~https://github.com/SatelliteQE/nailgun/pull/508~~

~~https://github.com/Katello/katello/pull/7560~~
~~https://github.com/Katello/katello/pull/7492~~

---

I wasn't going to raise this until the above PRs were merged, but it seems the Nailgun ones, in particular, are taking a while. Anyone interested in testing this is welcome to use https://github.com/sean797/nailgun/tree/mix which has the 4 commits cherry-picked.

### CODE REVIEW

**Comment by sean797** [comment]
Depending PRs merged and tests passed :fireworks:

File: module_utils/ansible_nailgun_cement.py [hunk 1]
```diff
@@ -229,12 +233,12 @@ def update_fields(new, old, fields):
 
 
 # Common functionality to manipulate entities
-def naildown_entity_state(entity_class, entity_dict, entity, state, module):
-    changed, _ = naildown_entity(entity_class, entity_dict, entity, state, module)
+def naildown_entity_state(entity_class, entity_dict, entity, state, module, check_missing=[]):
```
Comment (mdellweg) [inline module_utils/ansible_nailgun_cement.py]: Mutable objects as default Argumnents are almost never, what you expect. Please use` None` here.

Comment (sean797) [reply module_utils/ansible_nailgun_cement.py]: I did it as it saves me having to check if it's iterable later, anyway updated.

**Comment by sean797** [comment]
Thanks @mdellweg

**Pull request merged**